app/_hub/kong-inc/prometheus/index.md

```diff
@@ -81,6 +79,8 @@ dashboard: [https://grafana.com/dashboards/7424](https://grafana.com/dashboards/
 data plane nodes is exported to control plane.
 - **Enterprise License Information**: The {{site.base_gateway}} license expiration date, features and
 license signature. Those metrics are only exported on {{site.base_gateway}}.
+- **DB Entity Count**: A gauge metric that measures the current number of
+    database entities (**Enterprise-Only**).
 
```

Comment:
Should there also be an error count metric (kong_db_entity_count_errors)?

Reply:
I left it out because it's kind of a meta-metric, and this section seems like it's more of a general feature overview rather than an exhaustive reference. Though, I have no qualms about putting it in here if you think that's the way to go.

Reply:
You're right, none of these metric descriptions actually have the specific metric names, so that makes sense. Both should be covered with the sample output anyway.